trunk/SUAVE/Components/Energy/Networks/__init__.py

```diff
@@ -4,19 +4,20 @@
 # at each iteration to calculate thrust and a mass flow rate.
 # @ingroup Components-Energy
 
-from .Solar                                   import Solar
-from .Ducted_Fan                              import Ducted_Fan
-from .Battery_Ducted_Fan                      import Battery_Ducted_Fan 
-from .Turbofan                                import Turbofan
-from .Turbojet_Super                          import Turbojet_Super
-from .Solar_Low_Fidelity                      import Solar_Low_Fidelity
-from .Battery_Ducted_Fan                      import Battery_Ducted_Fan
-from .Internal_Combustion_Propeller           import Internal_Combustion_Propeller
-from .Lift_Cruise                             import Lift_Cruise
-from .Serial_Hybrid_Ducted_Fan                import Serial_Hybrid_Ducted_Fan
-from .Vectored_Thrust                         import Vectored_Thrust
-from .Propulsor_Surrogate                     import Propulsor_Surrogate
-from .Battery_Propeller                       import Battery_Propeller
-from .Ramjet                                  import Ramjet
-from .Scramjet                                import Scramjet
-from .Liquid_Rocket                           import Liquid_Rocket
+from .Solar                                        import Solar
+from .Ducted_Fan                                   import Ducted_Fan
+from .Battery_Ducted_Fan                           import Battery_Ducted_Fan 
+from .Turbofan                                     import Turbofan
+from .Turbojet_Super                               import Turbojet_Super
+from .Solar_Low_Fidelity                           import Solar_Low_Fidelity
+from .Battery_Ducted_Fan                           import Battery_Ducted_Fan
+from .Internal_Combustion_Propeller                import Internal_Combustion_Propeller
+from .Lift_Cruise                                  import Lift_Cruise
+from .Serial_Hybrid_Ducted_Fan                     import Serial_Hybrid_Ducted_Fan
+from .Vectored_Thrust                              import Vectored_Thrust
+from .Propulsor_Surrogate                          import Propulsor_Surrogate
+from .Battery_Propeller                            import Battery_Propeller
+from .Ramjet                                       import Ramjet
+from .Scramjet                                     import Scramjet
+from .Liquid_Rocket                                import Liquid_Rocket
+from .Internal_Combustion_Propeller_Constant_Speed import Internal_Combustion_Propeller_Constant_Speed

```

Comment:
What do you mean by constant speed? like constant rpm? Also is this another type of network or just a variant of the ICE network. If so, we can look to merge the two

Reply:
Constant speed is constant RPM. Constant Speed is the industry accepted term for constant RPM. The two methods rely on different engine calcs. So basically the networks are backwards of each other, CS goes prop->engine, Fixed pitch goes engine->prop. These networks are different enough I believe they should exist separately, and there is a clear need for both.